ehitchhiking-ui/src/app/shared/components/menu/menu.component.sass

```diff
@@ -1,33 +1,47 @@
-@import "../../../../styles/fonts-constants"
-@import "../../../../styles/colour-constants"
+@import "../../../../styles.sass"
 
 .sidenav-container
   height: 100%
   position: fixed
   top: 0
   background-color: $white
+
   .header
     display: flex
-    justify-content: space-around
-    .header-span
-      font: 100% $app-font
-    .settings
-      font-size: 2rem
-      margin-top: 1rem
-  .material-icons
-    font-size: $headline
-    margin: 0.5rem
+    align-content: center
+    justify-content: space-between
+    margin-top: 1rem
+    padding-right: 0.5rem
+    span 
```

Comment:
slow selectors, and I think angular style processor does not help in such cases.

Reply:
done